microcloud/cmd/microcloud/main_init.go

```diff
@@ -417,10 +418,21 @@ func AddPeers(sh *service.Handler, systems map[string]InitSystem) error {
 // setupCluster Bootstraps the cluster if necessary, adds all peers to the cluster, and completes any post cluster
 // configuration.
 func setupCluster(s *service.Handler, systems map[string]InitSystem) error {
-	_, bootstrap := systems[s.Name]
+	bootstapSystem, bootstrap := systems[s.Name]
 	if bootstrap {
 		fmt.Println("Initializing a new cluster")
 		err := s.RunConcurrent(true, false, func(s service.Service) error {
+			if s.Type() == types.MicroCeph && bootstapSystem.MicroCephNetworkSubnet != "" {
+				// When choosing a dedicated ceph network,
+				// make the public and cluster traffic flow through the same interfaces.
+				microCephBootstrapConf := map[string]string{
+					"PublicNet":  bootstapSystem.MicroCephNetworkSubnet,
+					"ClusterNet": bootstapSystem.MicroCephNetworkSubnet,
+				}
+
+				s.SetConfig(microCephBootstrapConf)
+			}
+
 			err := s.Bootstrap(context.Background())
```

Comment:
I wonder about the address the microcephd binds to; it looks to me that this would choose the same address as the other microcloud services right? Should this be changed to choose an address from the MicroCephNetworkSubnet?

Reply:
@sabaini Indeed, you're right. When `NewCephService` is called, the MicroCloud's internal network IP address of the bootstraping cluster member is chosen for microcephd to bind to. If a public subnet for Ceph has been specified, we should choose an address available on this subnet on this cluster member.

Reply:
I'm not so familiar with where the boundary is between the cluster and public networks of microceph, and microcluster's network, but judging from `microceph cluster bootstrap` I believe you're right @gabrielmougard.

Assuming I have 3 network interfaces: 
- `10.0.0.100/24` (default)
- `20.0.0.100/24`
- `30.0.0.100/24`

And I run `microceph cluster bootstrap --cluster-network `20.0.0.100/24` --public-network `30.0.0.100/24`, then the resulting microcluster address displayed in `microceph cluster list` is still the default `10.0.0.100:7443`.

So the internal microcluster network should still be propagated to microceph from `NewCephService`.